Backoffice/EventSubscriber/BlockTransverseCacheSubscriber.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\EventSubscriber;
+
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use OpenOrchestra\ModelInterface\NodeEvents;
+use OpenOrchestra\ModelInterface\Event\NodeEvent;
+use OpenOrchestra\DisplayBundle\Manager\CacheableManager;
+use OpenOrchestra\BaseBundle\Manager\TagManager;
+
+/**
+ * Class BlockTransverseCacheSubscriber
+ */
+class BlockTransverseCacheSubscriber implements EventSubscriberInterface
+{
+    protected $cacheableManager;
+    protected $tagManager;
+
+    /**
+     * @param CacheableManager $cacheableManager
+     * @param TagManager       $tagManager
+     */
+    public function __construct(CacheableManager $cacheableManager, TagManager $tagManager)
+    {
+        $this->cacheableManager = $cacheableManager;
+        $this->tagManager = $tagManager;
+    }
+
+    /**
+     * Triggered when a node status changes
+     *
+     * @param NodeEvent $event
+     */
+    public function invalidateNodeWithBlockTransverseTag(NodeEvent $event)
```

Comment:
@alavieille : ``invalidateNodeWithTransverseBlockTag`` instead of ``invalidateNodeWithBlockTransverseTag``